### PR TITLE
support unified read pool

### PR DESCRIPTION
**Pull request by sticnarf**
Signed-off-by: Yilin Chen <sticnarf@gmail.com>

This PR adds configuration and grafana pannels for unified read pool (which is introduced in TiKV 4.0).

### CODE REVIEW

**Comment by sticnarf** [comment]
This PR changes the template to use `for` to generate the sub-sections. Please check if there is any problem. Thanks.  @liubo0127

File: roles/tikv/templates/tikv.toml.j2 [hunk 1]
```diff
@@ -9,14 +9,17 @@
 {{ item }} = {{ value | to_json }}
 {% endfor %}
 
-[readpool.storage]
-{% for item, value in tikv_conf.readpool.storage | dictsort -%}
+[readpool]
```
Comment (liubo0127) [inline roles/tikv/templates/tikv.toml.j2]:
```suggestion
```

Comment (sticnarf) [reply roles/tikv/templates/tikv.toml.j2]: There is a field `unify-read-pool` under the readpool section.

Comment (liubo0127) [reply roles/tikv/templates/tikv.toml.j2]: If `[readpool]` is not deleted, the `[readpool]` item will be generated in the `tikv.toml`, but this configuration is not supported.

Comment (sticnarf) [reply roles/tikv/templates/tikv.toml.j2]: I think it is supported in latest TiKV now.

Comment (liubo0127) [reply roles/tikv/templates/tikv.toml.j2]: Reference to [#tikv.toml](https://github.com/tikv/tikv/blob/master/etc/config-template.toml), there is not `[readpool]` configuration.

**Comment by sticnarf** [comment]
PTAL again, thanks! @liubo0127

**Comment by liubo0127** [comment]
LGTM, and I will test it.

**Comment by sticnarf** [comment]
@liubo0127 Gentle ping. I wish this could release with 4.0.0-beta.1 :)

**Pull request merged**